packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -6,6 +6,7 @@
 /// @docImport 'time_picker.dart';
 library;
 
+import 'dart:io';
```

Comment:
I believe you've just made the date picker not work on Web (it doesn't have `dart:io` but `dart:html` instead). You added this import to use `Platform.isIOS`. Please use `defaultTargetPlatform` instead, [according to its docs](https://api.flutter.dev/flutter/foundation/defaultTargetPlatform.html).

Reply:
@creatorpiyush there are still `dart:io` imports in the files, after 432cd7d

Reply:
Fixed now